django/contrib/gis/geometry.py

```diff
@@ -10,7 +10,8 @@
     r"^(SRID=(?P<srid>\-?[0-9]+);)?"
     r"(?P<wkt>"
     r"(?P<type>POINT|LINESTRING|LINEARRING|POLYGON|MULTIPOINT|"
-    r"MULTILINESTRING|MULTIPOLYGON|GEOMETRYCOLLECTION)"
+    r"MULTILINESTRING|MULTIPOLYGON|GEOMETRYCOLLECTION|CIRCULARSTRING|SURFACE|TIN|"
+    r"POLYHEDRALSURFACE|TRIANGLE|CURVE|MULTISURFACE|MULTICURVE|COMPOUNDCURVE)"
```

Comment:
Should we also add `CURVEPOLYGON`?

Reply:
Yes. That looks right.

I'm away for a ~week now so will push edits when I get back. (Happy too if you want to push it over the line in the meantime)

Reply:
Pushed edits.